python/love/commander/salinfo.py

```diff
@@ -42,6 +41,7 @@ def create_app(*args, **kwargs):
         available_component_names = available_component_names[
             : kwargs.get("remotes_len_limit")
         ]
+
     salinfos = {}
```

Comment:
In English the word `info` seldomly is used in plural form. A better name would be `salinfo_list`. Feel free to ignore this comment completely!

Reply:
Nice! I'll apply this suggestion too :+1:.